### PR TITLE
Added Portuguese translation

### PR DESCRIPTION
**Pull request by VD171**

### CODE REVIEW

**Comment by aviraxp** [comment]
@VD171 啊这

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    <string name="module_release_no_more">Não há mais versões</string>
+    <string name="repo_load_failed">Falha ao carregar repositório de módulos: %s</string>
+
+    <string name="app_destroyed">Este app foi destruído, certifique-se de que você baixou da fonte oficial.</string>
+    <string name="outdated_manager">A versão do app do LSPosed e do módulo do LSPosed são inconsistentes. Por favor, reinstale a versão correspondente.</string>
+    <string name="lsposed_not_active">LSPosed não está instalado ou ativo neste momento.</string>
+    <string name="verbose_log_not_avaliable">O registro detalhado não está disponível, se você apenas ativou sem reiniciar, tente novamente após reiniciar.</string>
+    <string name="NotInstall">Não instalado</string>
+    <string name="NotInstallDetail">LSPosed não está instalado</string>
+    <string name="translators"><![CDATA[<a href="https://github.com/LSPosed/LSPosed">LSPosed</a>]]></string>
```
Comment (aviraxp) [inline app/src/main/res/values-pt/strings.xml:177]: Change it to you?

Comment (vvb2060) [inline app/src/main/res/values-pt/strings.xml:177]: Your name and contact information

**Review by aviraxp — APPROVED** [review]

File: app/src/main/res/values-pt/strings.xml (hunk 1 above)
Comment (VD171) [reply app/src/main/res/values-pt/strings.xml]: Done. Thanks

**Pull request merged**